packages/admin/admin-rte/src/core/Controls/SpecialCharactersControls.tsx

```diff
@@ -13,7 +14,16 @@ function SpecialCharactersControls(props: IControlProps) {
         return null;
     }
 
-    return <ButtonGroup>{supportedThings.includes("non-breaking-space") && <NonBreakingSpaceToolbarButton {...props} />}</ButtonGroup>;
+    if (!supportedThings.includes("soft-hyphen")) {
```

Comment:
Please also check for non-breaking-space here. If neither of them is supported, don't show the controls

Reply:
It is checked in line 13. Am I missing something?

Reply:
Oh, sorry, those lines weren't visible in the diff. You could merge the two ifs however

Reply:
On that note, it is not necessary to check the existence of the supportedThings below in the returned ButtonGroup too, is it?

Reply:
Yes it is, because there may be only one supported thing, not both